handbook/company/open-positions.yml

```diff
@@ -144,6 +144,25 @@
     - ✍ You have the ability to effectively influence key stakeholders, from senior executives to day-to-day engineering contacts, and drive Fleet's value with them
     - 🧬 You care about delivering an outstanding customer experience and advocating for the customer's needs within Fleet
     - ➕ Bonus: You are comfortable with concepts like security, APIs, and DevOps
+- jobTitle: 🐋 Sales Development Representative (SDR) 
+  department: Customers
+  hiringManagerName: Alex Mitchell
+  hiringManagerLinkedInUrl: https://www.linkedin.com/in/alexandercmitchell/
+  hiringManagerGithubUsername: alexmitchelliii
+  As someone on the front end of the sales journey, it is important to stay focused and organized in your responsibilities.: |
+What you’ll be doing: |
+    - 🎯 Prospect and execute outreach maneuvers working closely with your AE (Account Executive) to create opportunities
+    - 🧰 Maintain your tools. You have a tech-stack for a reason - to help you thrive. So update your CRM, reply to emails, set and complete tasks, stay organized and you’ll crush it
+    - 🧢 You’re more than a title. You’re a Fleetie. Learn and help with Marketing, BizOps, Product management and Engineering teams
+Who we’re looking for? 👀 : |
+    - 🤓 No need to be an expert, but remember the more you learn the more you earn
+    - 🦉Got experience? Great, if not, who cares. The real question is are you fearless with a ‘never fail, only learn’ attitude
```

Comment:
While I understand what we're trying to say, I don't think this is how we want to say it.